["    @Test\n    public void test1() throws Throwable {\n        if (debug)\n            System.out.format(\"%n%s%n\", \"RegressionTest0.test1\");\n        org.apache.pulsar.functions.instance.InstanceConfig instanceConfig0 = null;\n        org.apache.pulsar.functions.instance.AuthenticationConfig authenticationConfig1 = null;\n        // The following exception was thrown during execution in test generation\n        try {\n            java.util.List<java.lang.String> strList5 = org.apache.pulsar.functions.runtime.RuntimeUtils.getGoInstanceCmd(instanceConfig0, authenticationConfig1, \"\", \"\", false);\n            org.junit.Assert.fail(\"Expected exception of type java.lang.NullPointerException; message: Cannot invoke \\\"org.apache.pulsar.functions.instance.InstanceConfig.getClusterName()\\\" because \\\"instanceConfig\\\" is null\");\n        } catch (java.lang.NullPointerException e) {\n            // Expected exception.\n        }\n    }\n", "    @Test\n    public void test2() throws Throwable {\n        if (debug)\n            System.out.format(\"%n%s%n\", \"RegressionTest0.test2\");\n        org.apache.pulsar.functions.instance.InstanceConfig instanceConfig0 = null;\n        org.apache.pulsar.functions.instance.AuthenticationConfig authenticationConfig1 = null;\n        // The following exception was thrown during execution in test generation\n        try {\n            java.util.List<java.lang.String> strList5 = org.apache.pulsar.functions.runtime.RuntimeUtils.getGoInstanceCmd(instanceConfig0, authenticationConfig1, \"hi!\", \"hi!\", true);\n            org.junit.Assert.fail(\"Expected exception of type java.lang.NullPointerException; message: Cannot invoke \\\"org.apache.pulsar.functions.instance.InstanceConfig.getClusterName()\\\" because \\\"instanceConfig\\\" is null\");\n        } catch (java.lang.NullPointerException e) {\n            // Expected exception.\n        }\n    }\n", "    @Test\n    public void test3() throws Throwable {\n        if (debug)\n            System.out.format(\"%n%s%n\", \"RegressionTest0.test3\");\n        org.apache.pulsar.functions.instance.InstanceConfig instanceConfig0 = null;\n        org.apache.pulsar.functions.instance.AuthenticationConfig authenticationConfig1 = null;\n        // The following exception was thrown during execution in test generation\n        try {\n            java.util.List<java.lang.String> strList5 = org.apache.pulsar.functions.runtime.RuntimeUtils.getGoInstanceCmd(instanceConfig0, authenticationConfig1, \"hi!\", \"\", true);\n            org.junit.Assert.fail(\"Expected exception of type java.lang.NullPointerException; message: Cannot invoke \\\"org.apache.pulsar.functions.instance.InstanceConfig.getClusterName()\\\" because \\\"instanceConfig\\\" is null\");\n        } catch (java.lang.NullPointerException e) {\n            // Expected exception.\n        }\n    }\n", "    @Test\n    public void test4() throws Throwable {\n        if (debug)\n            System.out.format(\"%n%s%n\", \"RegressionTest0.test4\");\n        org.apache.pulsar.functions.instance.InstanceConfig instanceConfig0 = null;\n        org.apache.pulsar.functions.instance.AuthenticationConfig authenticationConfig1 = null;\n        // The following exception was thrown during execution in test generation\n        try {\n            java.util.List<java.lang.String> strList5 = org.apache.pulsar.functions.runtime.RuntimeUtils.getGoInstanceCmd(instanceConfig0, authenticationConfig1, \"\", \"hi!\", false);\n            org.junit.Assert.fail(\"Expected exception of type java.lang.NullPointerException; message: Cannot invoke \\\"org.apache.pulsar.functions.instance.InstanceConfig.getClusterName()\\\" because \\\"instanceConfig\\\" is null\");\n        } catch (java.lang.NullPointerException e) {\n            // Expected exception.\n        }\n    }\n", "    @Test\n    public void test5() throws Throwable {\n        if (debug)\n            System.out.format(\"%n%s%n\", \"RegressionTest0.test5\");\n        org.apache.pulsar.functions.instance.InstanceConfig instanceConfig0 = null;\n        org.apache.pulsar.functions.instance.AuthenticationConfig authenticationConfig1 = null;\n        // The following exception was thrown during execution in test generation\n        try {\n            java.util.List<java.lang.String> strList5 = org.apache.pulsar.functions.runtime.RuntimeUtils.getGoInstanceCmd(instanceConfig0, authenticationConfig1, \"\", \"\", true);\n            org.junit.Assert.fail(\"Expected exception of type java.lang.NullPointerException; message: Cannot invoke \\\"org.apache.pulsar.functions.instance.InstanceConfig.getClusterName()\\\" because \\\"instanceConfig\\\" is null\");\n        } catch (java.lang.NullPointerException e) {\n            // Expected exception.\n        }\n    }\n", "    @Test\n    public void test6() throws Throwable {\n        if (debug)\n            System.out.format(\"%n%s%n\", \"RegressionTest0.test6\");\n        org.apache.pulsar.functions.instance.InstanceConfig instanceConfig0 = null;\n        org.apache.pulsar.functions.instance.AuthenticationConfig authenticationConfig1 = null;\n        // The following exception was thrown during execution in test generation\n        try {\n            java.util.List<java.lang.String> strList5 = org.apache.pulsar.functions.runtime.RuntimeUtils.getGoInstanceCmd(instanceConfig0, authenticationConfig1, \"hi!\", \"\", false);\n            org.junit.Assert.fail(\"Expected exception of type java.lang.NullPointerException; message: Cannot invoke \\\"org.apache.pulsar.functions.instance.InstanceConfig.getClusterName()\\\" because \\\"instanceConfig\\\" is null\");\n        } catch (java.lang.NullPointerException e) {\n            // Expected exception.\n        }\n    }\n", "    @Test\n    public void test7() throws Throwable {\n        if (debug)\n            System.out.format(\"%n%s%n\", \"RegressionTest0.test7\");\n        org.apache.pulsar.functions.instance.InstanceConfig instanceConfig0 = null;\n        org.apache.pulsar.functions.instance.AuthenticationConfig authenticationConfig1 = null;\n        // The following exception was thrown during execution in test generation\n        try {\n            java.util.List<java.lang.String> strList5 = org.apache.pulsar.functions.runtime.RuntimeUtils.getGoInstanceCmd(instanceConfig0, authenticationConfig1, \"hi!\", \"hi!\", false);\n            org.junit.Assert.fail(\"Expected exception of type java.lang.NullPointerException; message: Cannot invoke \\\"org.apache.pulsar.functions.instance.InstanceConfig.getClusterName()\\\" because \\\"instanceConfig\\\" is null\");\n        } catch (java.lang.NullPointerException e) {\n            // Expected exception.\n        }\n    }\n", "    @Test\n    public void test8() throws Throwable {\n        if (debug)\n            System.out.format(\"%n%s%n\", \"RegressionTest0.test8\");\n        org.apache.pulsar.functions.instance.InstanceConfig instanceConfig0 = null;\n        org.apache.pulsar.functions.instance.AuthenticationConfig authenticationConfig1 = null;\n        // The following exception was thrown during execution in test generation\n        try {\n            java.util.List<java.lang.String> strList5 = org.apache.pulsar.functions.runtime.RuntimeUtils.getGoInstanceCmd(instanceConfig0, authenticationConfig1, \"\", \"hi!\", true);\n            org.junit.Assert.fail(\"Expected exception of type java.lang.NullPointerException; message: Cannot invoke \\\"org.apache.pulsar.functions.instance.InstanceConfig.getClusterName()\\\" because \\\"instanceConfig\\\" is null\");\n        } catch (java.lang.NullPointerException e) {\n            // Expected exception.\n        }\n    }\n"]